crates/node-bindings/src/geth.rs

```diff
@@ -132,8 +132,8 @@ impl GethInstance {
             err_reader.read_line(&mut line).map_err(GethInstanceError::ReadLineError)?;
 
             // geth ids are truncated
-            let truncated_id = hex::encode(&id.0[..8]);
-            if line.contains("Adding p2p peer") && line.contains(&truncated_id) {
+            let truncated_id = &id[0..8];
```

Comment:
this panics if too short

Reply:
thanks for the tip, then I change it to get the first 16bytes instead.